### DISCUSSION TITLE
How to ignore parent styles for md files?

### DISCUSSION THREAD

**Discussion by alperunal** [nuxt/content#1382] [Q&A]
Hello, 

I've added base CSS file to reset base styles to css array in nuxt.config. 
They also affect styles of the markdown files which located in content.
Is there any way to ignore the parent styles in md files?

**Comment by ManasMadrecha** [comment]
Don't add your CSS inside `nuxt.config`. 

In the `app.vue`'s `style` section, you can add 

```html
<style lang="scss">
body :not(.content) {
  /* Base styles here onwards */
}
</style>
```

And, in whichever `page` you use the Content Module's components to display md files, add `content` class to them.
